examples/getting-started-app/README.md

```diff
@@ -89,7 +94,7 @@ on the following files and directories:
    file of our project. In this tutorial, we discuss only a small part of it
    which deals with project dependencies.
 
-## Launching the project
+## First launching the project
```

Comment:
Consider "First launch" without "the project" part. Alternatively "First launch of the project"